x-pack/plugins/lens/public/indexpattern_datasource/operations/definitions/terms/values_range_input.tsx

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { useState } from 'react';
+import { useDebounce } from 'react-use';
+import { i18n } from '@kbn/i18n';
+import { EuiRange } from '@elastic/eui';
+
+type PropType<C> = C extends React.ComponentType<infer P> ? P : unknown;
+// Add ticks to EuiRange component props
+const FixedEuiRange = (EuiRange as unknown) as React.ComponentType<
```

Comment:
Can you submit a PR to Eui to fix this?

Reply:
Now looking at this code, I think it should already be fixed so I'll just remove it.